Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py

```diff
@@ -378,6 +397,7 @@ def generate_single_command_section(index, cmd, example_dict):
         'command_hr': cmd['name'],
         'command': cmd['name'],
         'command_description': cmd.get('description', ' '),
+        'permissions': get_permissions_entry(),
```

Comment:
pass a boolean here to indicate if the permission section needs to be inserted and then you can simply do:
`'permissions': PERMISSIONS_PER_COMMAND if should_include_permissions else '',`